src/partials.ts

```diff
@@ -6,23 +6,28 @@ import { resolve, parse } from 'path';
 /**
  * Registers each HTML file in a directory as Handlebars partial
  */
-export async function registerPartials(directoryPath: string): Promise<void> {
-  let dir: Dir;
+export async function registerPartials(directoryPath: string | Array<string>): Promise<void> {
+  // Convert to array
+  directoryPath = [].concat(directoryPath);
```

Comment:
I'd rather we not mutate an argument binding; this might be a better approach here instead:

```suggestion
  const pathArray: Array<string> = Array.isArray(directoryPath) ? directoryPath : [directoryPath];
```

Then use `pathArray` in the `for await` loop that iterates over all of the paths.